polls/models.py

```diff
@@ -83,3 +83,38 @@ class Annotation(models.Model):
 
     def __str__(self):
         return f"Annotation by {self.user.username}"
+
+
+class ExperimentType(models.Model):
+    type_choices = (
+        ("2AFC", "2AFC"),
+        ("A/B", "A/B"),
+    )
+    type = models.CharField(max_length=10, choices=type_choices)
+
+    def __str__(self):
+        return f"{self.type}"
+
+
+class Experiment(models.Model):
+    name = models.CharField(max_length=50)
+    experiment_type = models.ForeignKey(ExperimentType, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class ExperimentTypeTaskPresentation(models.Model):
+    task_presentation = models.CharField(max_length=50)
+    experiment_type = models.ForeignKey(ExperimentType, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return f"{self.task_presentation}"
+
+
+class ExperimentTypeAnnotation(models.Model):
```

Comment:
Why are ExperimentTypeTaskPresentation and ExperimentTypeAnnotation different tables? Why not combine them?

Reply:
I don't see any benefit to having them separate and logically they belong in the same row,  I think

Reply:
I know it's spec'ed that way but I don't see any benefit to it, so tell me if you see any reason they should be together. Maybe ExperimentType and ExperimentTypeTaskPresentation and ExperimentTypeAnnotation should be together in ExperimentType

Reply:
I'm in support of this, it's enough to have only ExperimentType with the rest being rows in the db

Reply:
I think we can just use `ArrayField(models.CharField(max_length=100))` and have `annotations` and `task_presentations` in Experiment model.

Reply:
Yes I was also thinking about suggesting to combine these tables. Will do it.

Reply:
> I think we can just use `ArrayField(models.CharField(max_length=100))` and have `annotations` and `task_presentations` in Experiment model.

I have added the ArrayField but it seems to work only with postgres. It will break when the tests will execute with sqllite

Reply:
I will be working on normalizing the database to avoid using ArrayFields.